app/assets/javascripts/wizard.js

```diff
@@ -355,6 +355,11 @@
     e.preventDefault();
     expandFieldset();
   });
-  
+
+  // show unrolled details tag if they've come to the page with a known location hash
+  var hash = window.location.hash;
+  if(hash == '#other_means' || hash == '#refer-to-ico' || hash == '#internal_review') {
```

Comment:
This won't work for international reuses where there are different next actions.

Should be possible to look up whether there's a target that matches the `window.location.hash` before adding the flash so that we don't have to hard code them?

Reply:
ah, of course. I was going super simple and went too super simple. I'll have a think